content/pages/cookies/policies.json

```diff
@@ -9,7 +9,8 @@
         "languageHelp": "Language of the cookie policy",
         "updated": "Last updated on",
         "dateFormat": "MMMM dd, yyyy."
-      }
+      },
+      "slug": "/cookies"
```

Comment:
thought: The slug is the same for all policies of the same kind in the array. Think about moving it out of the array.